node/test/dirtests.ts

```diff
@@ -1826,7 +1826,7 @@ describe('Dir Operation Tests', function () {
 
         var worked: boolean = false;
         try {
-            tl.mv(sourceFile, destFile);
+            tl.mv(sourceFile, destFile, "-n");
```

Comment:
What's this for?

Reply:
Shelljs changed the default for mv from "-n" (don't override files) to "-f" (override files). This restores the idea behind the test. 

I checked in the tasks repo and no tasks are depending on this behavior.